packages/orbit-components/src/Inline/helpers.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+import { left } from "../utils/rtl";
+import type { Theme, ThemeProps } from "../defaultTheme";
+
+import type { SpacingToken, Position } from "./index";
+
+type Prop = "align" | "justify" | "spacing";
+
+export const formatCSS = (key: string, value: string): string => `${key}: ${value};`;
+
+const getAlign = align => {
+  if (align === "start") return "flex-start";
+  if (align === "end") return "flex-end";
+
+  return "center";
+};
+
+export const normalizeSpacing = (el: SpacingToken, theme: Theme): string => {
+  const tokens = {
+    none: "",
+    "xxx-small": theme.orbit.spaceXXXSmall,
+    "xx-small": theme.orbit.spaceXXSmall,
+    "x-small": theme.orbit.spaceXSmall,
+    small: theme.orbit.spaceSmall,
+    medium: theme.orbit.spaceMedium,
+    large: theme.orbit.spaceLarge,
+    "x-large": theme.orbit.spaceXLarge,
+    "xx-large": theme.orbit.spaceXXLarge,
+    "xxx-large": theme.orbit.spaceXXXLarge,
+  };
+
+  if (el !== "none") {
+    return `
+      margin-${left({ theme })}: -${tokens[el]};
+      margin-top: -${tokens[el]};
+
+      & > * {
+        margin-${left({ theme })}: ${tokens[el]};
+        margin-top: ${tokens[el]};
+      }
+    `;
+  }
+
+  return "";
+};
+
+type PropObject = { [key: Prop]: Position };
+
+// TODO: kinda weird, but it's well known problem in flow with Object.values
```

Comment:
nope, it's not related to `any` below, i used it blow, because i did not find any other option how to solve it. That's basically issue in Flow, TS does not have such problems

Reply:
<img width="742" alt="Screenshot 2020-10-30 at 00 11 50" src="https://user-images.githubusercontent.com/14054752/97621292-07161f80-1a23-11eb-85ae-9dd217830687.png">

the problem is with `mixed` value in `Flow`. I tried to type it, with generics and etc, nothing helped, you can try to play with it and maybe you'll find solution, there are a lot of similar issues when i searched it